client/src/components/UserPanel.js

```diff
@@ -58,12 +59,12 @@ function UserPanel(props) {
         />
       </Grid>
       <Grid item id="name" className={classes.item}>
-        <Typography className={classes.primaryText}>John Doe</Typography>
-        <Typography className={classes.secondaryText}>@johndoe</Typography>
+        <Typography className={classes.primaryText}>{name}</Typography>
+        <Typography className={classes.secondaryText}>@{name}</Typography>
```

Comment:
this should be username right?

Reply:
are we adding that field?

Reply:
yes I was going to bring that up during the sprint planning. We will have to refactor that sooner rather ran later.